techniques/failures/F15.html

```diff
@@ -19,32 +32,35 @@
       <section class="procedure"><h3>Procedure</h3>
          <ol>
             <li> Using the accessibility checker for the technology (or if
-                                    that is not available, inspect the code or test with an
+                                    that is not available, inspect the code using a browser's dev tools, or test with an
                                     assistive technology), check the controls to see if they support
-                                    the accessibility API. </li>
+                                    the accessibility API.</li>
          </ol>
       </section>
       <section class="results"><h3>Expected Results</h3>
          <ul>
             <li> If step #1 is false, then this failure condition applies and the
-                                    content fails this Success Criterion </li>
+                                    content fails this success criterion.</li>
          </ul>
       </section>
    </section><section id="related"><h2>Related Techniques</h2></section><section id="resources"><h2>Resources</h2>
       
          <ul>
+           <li>
+             <a href="https://accessibilityinsights.io/docs/windows/overview/">Accessibility Insights for Windows</a>
```

Comment:
the a11y insights for windows is perfectly fine to use if you want to check the UIA tree of how the HTML is exposed.  i included this link specifically because the previous tool (which was used to check msaa/ia2) is no longer being updated (but still available)

this is fine to leave and can be useful for those that know how to use it.